internal/server/server.go

```diff
@@ -33,6 +34,7 @@ func (s *Server) Init() *Server {
 		s.router = http.NewServeMux()
 	}
 
+	s.router.Handle("GET    /metrics", promhttp.Handler())
```

Comment:
[nit] Let's drop the whitespace between `GET` and `/metrics`, since it doesn't align with the lines below anyways.

[nit] Perhaps we could add something to `internal/metrics` so that we don't spread promhttp over the code base... mainly for cleanliness, it doesn't need to be anything more than
```go
func Handler() http.Handler {
  return promhttp.Handler()
}
```

Centralizing this into the metrics package might help user later on... 💭